src/mapping/query/function/mod.rs

```diff
@@ -127,37 +122,6 @@ macro_rules! build_signatures {
 
 // List of built-in functions.
 build_signatures! {
-    to_string => ToStringFn,
-    to_int => ToIntegerFn,
-    to_float => ToFloatFn,
-    to_bool => ToBooleanFn,
-    to_timestamp => ToTimestampFn,
-    parse_timestamp => ParseTimestampFn,
-    strip_whitespace => StripWhitespaceFn,
-    upcase => UpcaseFn,
-    downcase => DowncaseFn,
-    uuid_v4 => UuidV4Fn,
-    md5 => Md5Fn,
-    sha1 => Sha1Fn,
-    sha2 => Sha2Fn,
-    sha3 => Sha3Fn,
-    now => NowFn,
-    truncate => TruncateFn,
-    parse_json => ParseJsonFn,
-    format_timestamp => FormatTimestampFn,
-    contains => ContainsFn,
-    slice => SliceFn,
-    tokenize => TokenizeFn,
-    strip_ansi_escape_codes => StripAnsiEscapeCodesFn,
-    parse_duration => ParseDurationFn,
-    format_number => FormatNumberFn,
-    parse_url => ParseUrlFn,
-    starts_with => StartsWithFn,
-    ends_with => EndsWithFn,
-    round => RoundFn,
-    floor => FloorFn,
-    ceil => CeilFn,
-    parse_syslog => ParseSyslogFn,
     split => SplitFn,
```

Comment:
Is there a reason Split has been left here?

Reply:
No reason, It'll be removed in the clean-up PR which removes the old implementation.